prefect_databricks/jobs.py

```diff
@@ -923,7 +923,7 @@ async def jobs_run_now(
     | 401 | The request was unauthorized. |
     | 500 | The request was not handled correctly due to a server error. |
     """  # noqa
-    endpoint = "/2.1/jobs/run-now"  # noqa
+    endpoint = "2.1/jobs/run-now"  # noqa
```

Comment:
Is removing this slash necessary? If not, I'd recommend leaving it to be consistent with the other tasks in this module.
```suggestion
    endpoint = "/2.1/jobs/run-now"  # noqa
```